### ISSUE TITLE
LOS with other nodes (bug?)

### ISSUE THREAD

**Issue by krisken** [wind-project/wind#56]
When i go the page of one of my nodes (index.php?node=xxxx) and i click the button "Line of sight (LOS) with other nodes" a new screen appears.  So far, so good.

At the left side, I see the node i've selected.  At the right side i have to select a node myself.  So i click on "change".  Normally a list of all my nodes will appear, but now, i get a new window.  Exactly the same window when i click on "Line of sight (LOS) with other nodes".

I suppose this is a bug?

**Comment by spinza** [comment]
Seeing the same.  Will try to fix it.

**Comment by jammin84** [comment]
yeah, we just ignored that, and added a LOS to all AP's and another for all nodes .

Here is an example.
http://wafn.jolly.so/wind/index.php/nodes?node=18

I can submit code if the community is interested?

**Comment by jammin84** [comment]
Some further investigation into this matter.

The LOS picker works when you click change from an already working link.
ie go to a node > scroll down to links > click on link > displays plot > click change node. works

These are the points of difference in the urls from the working version mentioned above and the bugged method.

working change
http://wafn.jolly.so/wind/index.php/nodes?page=nodes&subpage=plot_link&a_node=18&b_node=10
opens url 
http://wafn.jolly.so/wind/index.php/pickup/nodes?object=form_nodes_plot_link.b_node

not working
http://wafn.jolly.so/wind/index.php/nodes/plot_link?a_node=18
opens url
http://wafn.jolly.so/wind/index.php/nodes/plot_link?object=form_nodes_plot_link.b_node

test (working minus b_node)
http://wafn.jolly.so/wind/index.php/nodes?page=nodes&subpage=plot_link&a_node=18
opens url (incorrectly)
http://wafn.jolly.so/wind/index.php/nodes?object=form_nodes_plot_link.b_node
should be
http://wafn.jolly.so/wind/index.php/pickup/nodes?object=form_nodes_plot_link.b_node

This should help relative to finding which code to resolve the issue.
